components/autogen/src/doc/FormatPage.vm

```diff
@@ -96,6 +96,11 @@ Utility: |$utilityRating|
 format, we are not able to distribute them to third parties.**
 #end
 
+#if ($options)
+Additional options are available for reading this format type, see
+:doc:`options` for information.
+#end
+
```

Comment:
Inverting lines 102 and lines 103 (i.e. putting the extra line in the conditional block) ad regenerating the format pages might fix the unnecessary blank lines additions for all formats.